src/main/resources/application.properties

```diff
@@ -343,6 +343,15 @@ alpine.oidc.team.synchronization=false
 # will most likely need to be configured.
 alpine.oidc.teams.claim=groups
 
+# Optional
+# Toggle the activation and exposition of H2 database web console.
+# The default value is false.
```

Comment:
A word or two about the security implications of enabling the console would be good to have.